content/tutorials/jwt-authentication.textile

```diff
@@ -273,15 +276,29 @@ The last part is to add the logic into this form. You'll instantiate Ably's Real
 ```[html]
 <script type="text/javascript">
 console.log('On login page now')
+
+
 function login(e) {
-    e.preventDefault();
-    console.log('Fetching JWT token from auth server')
-    const realtime = new Ably.Realtime({ authUrl: '/auth' });
-    realtime.connection.once('connected', function () {
-    console.log('Client connected to Ably using JWT')
-    alert("Client successfully connected to Ably using JWT auth")
-    })
-}
+  e.preventDefault();
+
+  const username = document.querySelector("#username").value;
+  const password = document.querySelector("#password").value;
+
+  if (!`${username}`.trim() || !`${password}`.trim()) {
+    return alert("Error! Require username and password");
+  }
+
+  // Send user details to the auth server
+  const authUrl = `/auth?clientId=${username}:${password}`;
+  const realtime = new Ably.Realtime({ authUrl });
+
+  console.log("Fetching JWT token from auth server", username, password);
+
+  realtime.connection.once("connected", () => {
+    const { tokenDetails } = realtime.auth;
+    console.log("Client connected to Ably using JWT", tokenDetails);
```

Comment:
"Client connected to the Ably platform using an Ably JWT"

Reply:
Success! You're connected with a JWT token.